lighthouse-core/lib/dependency-graph/simulator/simulator.js

```diff
@@ -141,9 +141,8 @@ class Simulator {
 
   /**
    * @param {Node} node
-   * @param {number} queuedTime
    */
-  _markNodeAsReadyToStart(node, queuedTime) {
```

Comment:
I'd prefer to keep this, it's pretty helpful when stepping through the debugger

Reply:
> I'd prefer to keep this, it's pretty helpful when stepping through the debugger

gotcha :)